### ISSUE TITLE
Component location rules problem when Component === All is selected

### ISSUE THREAD

**Issue by barnemax-pilotin** [Pilot-in/PiloPress#107]
**Describe the bug**
When a field group has location rule set to Components === All, the field group is displayed in any post type edit page.

**To Reproduce**
Blank Pilo'Press install : [https://www.loom.com/share/9c7e7e607a724ab99da392471528cf2a](https://www.loom.com/share/9c7e7e607a724ab99da392471528cf2a) (sorry for the laggy computer)

**Expected behavior**
The field group to be displayed only on the Component's edit page

The fix I used here is : 
`plugins\pilopress\includes\classes\components\class-components.php l.206`

```
global $current_screen;
$match = $current_screen->post_type === 'pip-components';
```
Basically checking if the current page is the edit page of components, maybe there is a more elegant way to do it.

**WordPress, Plugins**
WordPress version: 5.6
Pilo'Press version: 0.3.2.11
ACF Pro version:  5.9.3
ACFE version: 0.8.7.6

**Comment by damien-pilot-in** [comment]
You can make a pull request to fix this i think (since you fixed the issue yourself 😄)
